app/templates/app/institutions.html

```diff
@@ -15,14 +16,14 @@ <h5 class="card-title">{{ institution.name }}</h5>
                     </p>
                     <p class="card-text">
                         {% if institution.project_count == 0 %}
-                            no applicable project
+                            {% blocktrans %}no applicable project{% endblocktrans %}
                         {% else %}
-                            {{institution.project_count}} applicable projects
+                            {% blocktrans %}{{institution.project_count}} applicable projects{% endblocktrans %}
```

Comment:
This one requires plural handling, please.

Reply:
will do